app/controllers/likes_controller.rb

```diff
@@ -7,6 +9,22 @@ def create
     else
       flash[:error] = 'like was not added'
     end
-    redirect_to user_post_url
+    redirect_to user_post_path(@post)
   end
+
+  # load_and_authorize_resource
+
+  # def create
+  #   post = Post.find(params[:id])
+  #   like = current_user.likes.create(post: post)
+  #   like.save!
+
+  #   if like.save
+  #     flash[:success] = 'Post Liked!'
+  #     redirect_to user_post_path
+  #   else
+  #     flash.now[:error] = 'Like not added...'
+  #     redirect_to post_path(@post)
+  #   end
+  # end
```

Comment:
- [ ] I would recommend removing the commented code to make the codebase cleaner.